bin/main.rkt

```diff
@@ -144,6 +145,7 @@
       (error "Output signal must be specified as <name>:<bw>"))
     (verilog-module-out-signal (first splits))
     (verilog-module-out-bitwidth (string->number (second splits))))]
+ ["--output-smt-path" v "Specify the output of the SMT solver to a file." (output-smt-path v)]
```

Comment:
This path actually refers to the _directory_ where the files will be placed, not the file

Reply:
Oops yeah you're right, I'll change the line to reflect this

Reply:
See this thread, still unresolved